operator/auth/spire/log_wrapper.go

```diff
@@ -3,35 +3,37 @@
 
 package spire
 
-import "github.com/sirupsen/logrus"
+import (
+	"log/slog"
+)
 
 // spiffeLogWrapper is a log wrapper for the SPIRE client logs
 // the log levels of this library do not match those from Cilium
 // this will be used to convert the log levels.
 type spiffeLogWrapper struct {
-	log logrus.FieldLogger
+	log *slog.Logger
 }
 
 // newSpiffeLogWrapper returns a new spiffeLogWrapper
-func newSpiffeLogWrapper(log logrus.FieldLogger) *spiffeLogWrapper {
+func newSpiffeLogWrapper(log *slog.Logger) *spiffeLogWrapper {
 	return &spiffeLogWrapper{
 		log: log,
 	}
 }
 
 // Debugf logs a debug message
 func (l *spiffeLogWrapper) Debugf(format string, args ...interface{}) {
-	l.log.Debugf(format, args...)
+	l.log.Debug(format, args...)
 }
 
 // Infof logs an info message
 func (l *spiffeLogWrapper) Infof(format string, args ...interface{}) {
-	l.log.Infof(format, args...)
+	l.log.Info(format, args...)
 }
 
 // Warnf logs a warning message
 func (l *spiffeLogWrapper) Warnf(format string, args ...interface{}) {
-	l.log.Warnf(format, args...)
+	l.log.Warn(format, args...)
 }
```

Comment:
These will now panic if `len(args) % 2 != 0` as they expect `args` to be a set of key-value pairs. Should we instead convert these to use `fmt.Sprintf` to maintain the current behavior?

Reply:
I don't think panic will happen (i.e. some predefined string will be used), but it's good point to change to fmt.Sprintf :+1: